### PR TITLE
void-updates: update to 20221026

### PR DESCRIPTION
**Pull request by mdkcore0**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-libc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l (crossbuild)
  - i686-libc (crossbuild)

### CODE REVIEW

**Comment by classabbyamp** [comment]
there's a couple more commits that should probably make it into this update, but will require another tag

**Comment by mdkcore0** [comment]
looking on the void-updates repo there are 2 commits ahead of the 20210331 tag, I can convert this PR to draft and ask there for adding a new tag to include such commits, so we can have the latest code for the package

**Comment by mdkcore0** [comment]
> i'll ask on irc, it'll be faster

coo, thanks! (also I need to get back to the void channel on libera :))

**Comment by classabbyamp** [comment]
https://github.com/void-linux/void-updates/releases/tag/20220111

**Comment by mdkcore0** [comment]
> https://github.com/void-linux/void-updates/releases/tag/20220111

thanks, will build and test locally and update the pr, thanks!

**Comment by classabbyamp** [comment]
(fyi I edited my last message, had to re-tag)

**Comment by mdkcore0** [comment]
updated to 20221026

File: srcpkgs/void-updates/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-revision=2
+version=20221026
+revision=1
 build_style=gnu-makefile
 depends="git curl"
 short_desc="Void Linux package updates"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="ISC"
 homepage="https://github.com/void-linux/void-updates"
 distfiles="${homepage}/archive/${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/void-updates/template:11]:
```suggestion
 distfiles="https://github.com/void-linux/void-updates/archive/${version}.tar.gz"
```

**Pull request merged**